.vib/nats/goss/goss.yaml

```diff
@@ -0,0 +1,10 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../nats/goss/nats.yaml: {}
+  # Load scripts from .vib/common/goss/templates
+  ../../common/goss/templates/check-app-version-stdout.yaml: {}
```

Comment:
I think you are missing the `-no-shell-` substring here.

`.vib/common/goss/templates/check-app-version-no-shell-stdout.yaml`